runtime/store.go

```diff
@@ -48,6 +48,12 @@ func (t transientStoreService) OpenTransientStore(ctx context.Context) store.KVS
 	return newKVStore(sdk.UnwrapSDKContext(ctx).KVStore(t.key))
 }
 
+type failingStoreService struct{}
```

Comment:
is this type used anywhere?

Reply:
Nope, remaining of reverted changes. I split the PR in two as it contained unrelated changes.